chcemvediet/apps/anonymization/cron.py

```diff
@@ -0,0 +1,31 @@
+from django.core.files.base import ContentFile
+
+from poleno.cron import cron_job
+from poleno.attachments.models import Attachment
+from chcemvediet.apps.inforequests.models import Action
+
+from .models import AttachmentNormalization
+from . import content_types
+
+
+@cron_job(run_every_mins=1)
+def attachment_normalization():
+    attachments = Attachment.objects.attached_to(Action).filter(
+        attachmentnormalization__isnull=True
+    )[:10]
```

Comment:
Selectni iba jeden attachment. Samotna konverzia bude trvat dost dlho, takze za minutu ich 10 ani nestihneme. Takisto, ked jedna z konverzii failne, nech ostatne su v samostatnych tranzakciach. Jeden atachment neselectuj pomocou `[:1]` ale by na to malo byt nieco explicitnejsie. Potom pozor na to, ak ziaden taky attachment neexistuje.